pkg/options/auth_test.go

```diff
@@ -169,7 +162,8 @@ auth:
 			assert := assert.New(t)
 			options := &SubOptions{}
 			err := yaml.UnmarshalStrict([]byte(test.input), options)
-
+			jsn, _ := yaml.Marshal(test.expected)
+			fmt.Println(string(jsn))
```

Comment:
Please use `t.Log(...)` instead, or just remove it.

Reply:
I will have this removed. It doesn't serve any purpose here